doc-content/enterprise-only/cases/case-management-dynamic-tasks-proc.adoc

```diff
@@ -18,24 +12,17 @@ endif::[]
 * The IT Orders project is deployed to {KIE_SERVER}.
 * The Showcase application `.war` file has been deployed alongside {CENTRAL}.
 
-.Procedure 
-. With the IT Orders project deployed and running in the {KIE_SERVER}, in a web browser, navigate to the Showcase login page `http://localhost:8080/rhpam-case-mgmt-showcase/`. 
+.Procedure
+. With the *IT_Orders_New* project deployed and running in the {KIE_SERVER}, in a web browser, navigate to the Showcase login page `http://localhost:8080/rhpam-case-mgmt-showcase/`.
```

Comment:
In this context, to quickly get started with the showcase, I believe we want to encourage to use the out-of-box example, which is _IT_Orders_.

Reply:
Changed back to the sample